src/NuGet.Services.Staging.Manager/Startup.cs

```diff
@@ -41,12 +42,13 @@ public Startup(IHostingEnvironment hostingEnvironment)
                 .AddJsonFile(Path.Combine("Config", $"config.{hostingEnvironment.EnvironmentName}.json"))
                 .AddEnvironmentVariables();
 
-            Configuration = builder.Build();
-
             if (hostingEnvironment.IsEnvironment(_localEnvironmentName))
             {
                 builder.AddApplicationInsightsSettings(developerMode: true);
             }
+
+            var configBuild = builder.Build();

```

Comment:
I'm not sure what you mean here. Caching of what? Does this method get called more than once?

---
In reply to: [74161213](https://github.com/NuGet/NuGet.Staging/pull/69#discussion_r74161213) [](ancestors = 74161213)